### PR TITLE
CAT-2662 Added ActionBar and ActionMode title tests

### PR DESCRIPTION
**Pull request by LPeteR90**

### CODE REVIEW

**Comment by 84n4n4** [comment]
test this please

File: catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/actionbar/utils/ActionModeWrapper.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import org.catrobat.catroid.R;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+public final class ActionModeWrapper {
+	private ActionModeWrapper() { }
```
Comment (84n4n4) [inline catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/actionbar/utils/ActionModeWrapper.java]: same as above with the actionBarWrapper

File: catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/actionbar/utils/ActionModeWrapper.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	}
+
+	public ActionModeWrapper checkTitleMatches(String text) {
+		onView(withId(R.id.action_bar_title)).check(matches(withText(text)));
+		return this;
+	}
+
+	public ActionModeWrapper performGoBack() {
+		onView(withId(R.id.action_mode_close_button)).perform(click());
+		return this;
```
Comment (84n4n4) [inline catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/actionbar/utils/ActionModeWrapper.java]: here i wouldnt return anything since actionbar should be gone and nothing can be done anymore with this object.

File: catroid/src/main/java/org/catrobat/catroid/content/bricks/DroneMoveRightBrick.java [hunk 3]
```diff
@@ -61,12 +61,13 @@ private void initializeBrickFields(Formula durationInSeconds, Formula powerInPer
...
 	public void showFormulaEditorToEditFormula(View view) {
 		switch (view.getId()) {
-			case R.id.brick_drone_move_right_edit_text_second:
-				FormulaEditorFragment.showFragment(view, this, BrickField.DRONE_TIME_TO_FLY_IN_SECONDS);
-				break;
 			case R.id.brick_drone_move_right_edit_text_power:
 				FormulaEditorFragment.showFragment(view, this, BrickField.DRONE_POWER_IN_PERCENT);
 				break;
+			case R.id.brick_drone_move_right_edit_text_second:
+			default:
```
Comment (84n4n4) [inline catroid/src/main/java/org/catrobat/catroid/content/bricks/DroneMoveRightBrick.java]: stuff like this that doesnt have anything todo with the ticket should be a separate PR, or at least a separate commit

Comment (LPeteR90) [reply catroid/src/main/java/org/catrobat/catroid/content/bricks/DroneMoveRightBrick.java]: Hmmm... Don't know how that stuff got in here, didn't change anything there, will remove it from PR

**Comment by LPeteR90** [comment]
test this please

**Comment by 84n4n4** [comment]
please run espresso tests

**Comment by LPeteR90** [comment]
please run espresso tests

**Comment by LPeteR90** [comment]
please run espresso tests

**Comment by LPeteR90** [comment]
please run espresso tests
